common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -80,7 +81,8 @@ function () {
 
         if (value) {
             state.videoAccessibleMenu.setValue(value);
-            button.text(gettext(msg));
+            // button.text(gettext(msg));
+            downloadLink.text(gettext('Download transcript (' + msg + ')'));
```

Comment:
This usage of gettext won't work-- you can't concatenate strings like this.

You need to use a placeholder:

interpolate(gettext('Download transcript (%(msg)s)?'), { msg: msg }, true)

If you can think of a more descriptive name than "msg" for the variable, that would be helpful for the translators.